### PR TITLE
Adding Release Details to Tizen 7.0 M2 Public Release Note

### PR DESCRIPTION
**Pull request by safir-srbd**
Adding the release details to the Tizen 7.0 M2 Public Release Note (#1694)

### CODE REVIEW

**Comment by safir-srbd** [comment]
@mijin85cho Can you check if everything is in order?

File: docs/platform/release-notes/tizen-6-5-m2.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Release date: Oct. 31, 2021
 
 ## Release details
 
-- [Getting source code](http://review.tizen.org/git/) (Tizen 6.5 M2 source codes are under **tizen** branch.)
+- [Getting source code](http://review.tizen.org/git/) (Tizen 6.5 M2 source codes are under **tizen_6.5** branch.)
```
Comment (journey2w) [inline docs/platform/release-notes/tizen-6-5-m2.md:7]:
```suggestion
- [Getting source code](http://review.tizen.org/git/) (Tizen 7.0 M2 source codes are under **tizen_7.0** branch.)

```

File: docs/platform/release-notes/tizen-7-0-m2.md [hunk 2]
```diff
@@ -4,7 +4,8 @@ Release date: Oct. 31, 2022
 
 ## Release details
 
-- [Getting source code](http://review.tizen.org/git/) (Tizen 7.0 M1 source codes are under **tizen** branch.)
+- [Getting source code](http://review.tizen.org/git/) (Tizen 7.0 M2 source codes are under **tizen** branch.)
```
Comment (journey2w) [inline docs/platform/release-notes/tizen-7-0-m2.md:7]: Platform team already 7.0 branch out.

```suggestion
- [Getting source code](http://review.tizen.org/git/) (Tizen 7.0 M2 source codes are under **tizen_7.0** branch.)
````

**Review by mijin85cho — APPROVED** [review]
Thanks for the update.

**Comment by journey2w** [comment]
@safir-srbd 
I'm sorry but please update the branch name in tizen 7.0 m1 release notes toot.

**Comment by safir-srbd** [comment]
> @safir-srbd I'm sorry but please update the branch name in tizen 7.0 m1 release notes toot.

@journey2w Thank you for informing me. I am on it!

**Comment by safir-srbd** [comment]
@journey2w I have made the change you requested.

**Pull request merged**